sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -150,8 +179,7 @@ export class SenderImpl implements Sender {
   async send(message: SendableMessageInfo): Promise<void> {
     this._throwIfSenderOrConnectionClosed();
     throwTypeErrorIfParameterMissing(this._context.namespace.connectionId, "message", message);
-    const sender = MessageSender.create(this._context);
-    return sender.send(message);
+    return this._sender.send(message);
   }
 
   async sendBatch(messages: SendableMessageInfo[]): Promise<void> {
```

Comment:
Let's hide this variation from the exported user surface and rename the `sendBatch2` to `sendBatch`. In the next preview, we will be re-visiting the array case.

Reply:
Hidden for now.
I have some thoughts.. (Probably, for preview 2)
- sendMessage(Message)
- sendMessages(Message[])
- sendMessageBatch(MessageBatch)

or discard single message sending and allow just these two.
- send(Message[]) -> send(Message[])
- sendBatch(MessageBatch) -> sendMessageBatch(MessageBatch)